src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -707,3 +733,31 @@ def _get_metadata_counts(
                     df[column_name].isin(nicknames.index).sum()
                 )
     return metadata_df
+
+
+def update_guardian_duplication_row_counts(melted_df: pd.DataFrame) -> pd.DataFrame:
+    # Update guardian duplication row counts to make number of rows match anticipated
+    # conditional probability
+
+    # This will be the number of rows where a dependent is in the group (the denominator)
+    # for the conditional probability
+    guardian_duplication_group_row_counts = melted_df.loc[melted_df["column"] == "group_rows"]
+    guardian_duplication_group_row_counts = guardian_duplication_group_row_counts.rename(
+        columns={"column": "column_y", "value": "group_row_counts", "variable": "variable_y"}
+    )
+    # Remove these rows from so we do not output them later
+    melted_df = melted_df.loc[
+        melted_df.index.difference(guardian_duplication_group_row_counts.index)
+    ]
+    # Merge two dataframes
+    melted_df = melted_df.merge(
+        guardian_duplication_group_row_counts,
+        how="left",
+        on=["dataset", "state", "year", "noise_type", "number_of_rows"],
```

Comment:
Same as above - I'm not sure if any of these strings are containerized somewhere but if not they probably should be